server/backend/api/config/config.py

```diff
@@ -143,3 +144,16 @@ def get_allow_origins():
     allow_origins = [get_env_var("ALLOW_ORIGIN")] if is_on_aws() else ["*"]
     LOGGER.info(f"allow_origins -- {allow_origins}")
     return allow_origins
+
+
+def get_forest_client_api_token():
+    api_token = get_env_var("FC_API_TOKEN") if is_on_aws() \
+        else "4r0dn5wC0JTs6N1tdw6NDzpjBRQwEJ6u"  # Test env.
```

Comment:
Probably bad security - this token is now exposed to the entire internet.

Reply:
Initially I treated this token in TEST environment isn't needed to be 'secure'.
You finally caught me, security officer!!

Reply:
I removed the default TEST token now, officer... I will only share it with my team privately (if they give me a chocolate, I like chocolate)

Reply:
Token removed.
Encountered using token from GH report secret for the CI run. Regular terraform plan seems to be fine.
Resolved from CI yaml template.